hydro_cli/src/core/gcp.rs

```diff
@@ -142,7 +147,14 @@ impl LaunchedHost for LaunchedComputeEngine {
         drop(created_file);
 
         let mut channel = session.channel_session().await?;
-        channel.exec(binary_path.to_str().unwrap()).await?;
+        let binary_path_string = binary_path.to_str().unwrap();
+        let args_string = args
+            .iter()
+            .map(|s| shell_escape::unix::escape(Cow::Owned(s.clone())))
```

Comment:
Oh I guess if youre iterating refs instead of Strings.. does `Cow::Borrowed(&**s)` work?